torch/utils/hooks.py

```diff
@@ -34,3 +35,14 @@ def __enter__(self):
 
     def __exit__(self, type, value, tb):
         self.remove()
+
+
+def warn_if_has_hooks(tensor):
+    if tensor._backward_hooks:
+        for k in tensor._backward_hooks:
+            hook = tensor._backward_hooks[k]
+            if not hasattr(k, "_torch_unserializable"):
+                warnings.warn("backward hook {} on tensor will not be "
+                              "serialized.  If this is expected, you can "
+                              "set the attribute _torch_unserializable on "
```

Comment:
You meant set `_torch_unserializable` to `False`, right? Worth clarifying that.

Reply:
Also, names prefixed with underscores are really meant to be internal, so we should never recommend people to do something with them. Just make it `__torch_unserializable__` or `torch_unserializable`, or even better provide a decorator for marking functions unserializable:
```python
from torch.utils.hooks import unserializable_hook
```